plugins/node/opentelemetry-instrumentation-koa/examples/src/server.ts

```diff
@@ -2,11 +2,13 @@
 
 const api = require('@opentelemetry/api');
```

Comment:
in `client.ts` and `tracer.ts` we **import** like this:
```
import * as api from '@opentelemetry/api';
```

Can we align this too to make code styling consistent?